src/append/opentelemetry.rs

```diff
@@ -56,7 +56,7 @@ pub struct OpentelemetryLogBuilder {
     endpoint: String,
     protocol: Protocol,
     labels: Vec<(Cow<'static, str>, Cow<'static, str>)>,
-    layout: Option<Layout>,
+    layout: Option<Box<dyn Layout>>,
```

Comment:
Do we have better solution to write `Option<Box>`?

Especially on accepting an existing box rather than always boxing on configuring layout.